examples/devices/eiger.py

```diff
@@ -0,0 +1,101 @@
+# import asyncio
+from typing import Awaitable, Callable, Optional
+
+from aiohttp import web
+
+from tickit.adapters.httpadapter import HTTPAdapter
+from tickit.adapters.interpreters.endpoints.http_endpoint import HTTPEndpoint
+from tickit.adapters.servers.http import HTTPServer
+from tickit.core.adapter import ConfigurableAdapter
+from tickit.core.device import ConfigurableDevice, DeviceUpdate
+from tickit.core.typedefs import SimTime
+from tickit.utils.byte_format import ByteFormat
+from tickit.utils.compat.typing_compat import TypedDict
+
+
+class Eiger(ConfigurableDevice):
```

Comment:
Since this is currently a generic, shall we rename it to `HTTPDevice`, leave it as an example and start the eiger on a different branch?

Reply:
Sounds good to me.